WORKSPACE

```diff
@@ -19,4 +19,4 @@ git_repository(
 
 load("@build_bazel_rules_typescript//:setup.bzl", "ts_setup_workspace")
 
-ts_setup_workspace()
+ts_setup_workspace(default_tsconfig = "@angular_devkit//:tsconfig.json")
```

Comment:
FYI, this may have to change a bit, see https://github.com/bazelbuild/rules_typescript/issues/122